spec/features/start_game_spec.rb

```diff
@@ -0,0 +1,31 @@
+require './app'
+
+# The marketeer is presented with three choices
+# rock, paper and scissors
+
+feature '#play_game' do 
+  
+  scenario 'present rock, paper, scissor options' do 
+    visit '/'
+    fill_in :player, with: 'Shanice'
```

Comment:
Nice tests - use a web helper to shorten the code rather than having to write the login code every time